docs/sections/contributor_guide/index.rst

```diff
@@ -2,6 +2,8 @@ Contributor Guide
 =================
 
 .. toctree::
+   :maxdepth: 2
```

Comment:
This makes the tables of contents a bit more compact and IMO easier to read/use.